### PR TITLE
Use valid HTML in templates

### PR DESCRIPTION
**Pull request by KaelWD**
Vue components are *fragments*, not entire pages. You can't load an entire HTML document into them. 

I also cleaned up the templates' style a bit, they were kinda messy

### CODE REVIEW

**Comment by Maxhodges** [comment]
were you able to build? this breaks as invalid syntax in vs code. is my editor broke?
image

**Comment by KaelWD** [comment]
Did you use some sort of code formatter? That should all be on one line. 
image

**Comment by Maxhodges** [comment]
yeah Vetur, but it has 1.4M users
i'll report the issue to them, if you say it's fine syntax.
image

**Comment by KaelWD** [comment]
That should probably be a `url(...)` actually: https://developer.mozilla.org/en-US/docs/Web/CSS/%40import

**Comment by Maxhodges** [comment]
i'm not able to run your branch for some reason. still investigating
```
MaxBook-Pro:vuetify-storyboard-boilerplate maxhodges$ npm run storybook

> vwre@1.0.0 storybook /Users/maxhodges/code/vuetify-storyboard-boilerplate
> start-storybook -p 9001 -c .storybook

@storybook/vue v3.2.16

=> Loading custom .babelrc
=> Loading custom addons config.
=> Using default webpack setup based on "vue-cli".
 10% building modules 2/2 modules 0 activeevents.js:160
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE :::9001
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at Server._listen2 (net.js:1262:14)
    at listen (net.js:1298:10)
    at Server.listen (net.js:1376:9)
    at Function.listen (/Users/maxhodges/code/vuetify-storyboard-boilerplate/node_modules/express/lib/application.js:618:24)
    at Object.<anonymous> (/Users/maxhodges/code/vuetify-storyboard-boilerplate/node_modules/@storybook/vue/dist/server/index.js:161:27)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vwre@1.0.0 storybook: `start-storybook -p 9001 -c .storybook`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the vwre@1.0.0 storybook script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/maxhodges/.npm/_logs/2017-12-05T13_06_17_121Z-debug.log
```

it ran for you?

**Comment by Maxhodges** [comment]
ok that fixed it (url)...

**Comment by KaelWD** [comment]
Yeah it runs for me. That error means you already have something listening on port 9001

**Comment by Maxhodges** [comment]
`port 9001` derp!

**Comment by Maxhodges** [comment]
@KaelWD thanks a lot for cleaning this up. btw, I also enhanced the template so that it takes the items from an items prop. Now there are two stories with pass different items as props to the menu.

File: src/stories/Layout.vue [hunk 1]
```diff
@@ -7,63 +7,76 @@
...
-            </v-card>
-          </v-flex>
+        <v-container grid-list-md text-xs-center>
+    <v-layout row wrap>
+      <v-flex xs12>
+        <v-card dark color="primary">
+          <v-card-text class="px-0"><slot/></v-card-text>
+        </v-card>
+      </v-flex>
+    </v-layout></v-container>
```
Comment (KaelWD) [inline src/stories/Layout.vue]: Why'd you undo this? Half the point of this PR was fixing the terrible code style.

Comment (Maxhodges) [reply src/stories/Layout.vue]: I don't see any different. You mean the line break after `<slot/>`?

Comment (Maxhodges) [reply src/stories/Layout.vue]: removed
```
            <v-card dark color="primary">
              <v-card-text class="px-0">
                <slot/>
              </v-card-text>
            </v-card>
```

added
```
        <v-card dark color="primary">
          <v-card-text class="px-0"><slot/></v-card-text>
        </v-card>
```

Comment (Maxhodges) [reply src/stories/Layout.vue]: the template is taken straight from vuetify, so maybe we should fix the terrible code style there?

**Comment by Maxhodges** [comment]
>Vue components are fragments, not entire pages. You can't load an entire HTML document into them.

you can apparently. My storybook was working! Maybe not a best practice, but in storybook we are just focused on developing components outside of running the app. So the 'layout' is just a wrapper so which we can use to show a component within an application layout. We couldn't use this layout as-is in our app. It's just a development framework for designing, testing and sharing components, as they appear in various states. @KaelWD

**Comment by KaelWD** [comment]
"It works" doesn't mean it's a good idea.

File: src/stories/Layout.vue (hunk 1 above)
Comment (Maxhodges) [reply src/stories/Layout.vue]: @KaelWD did i overlook something you contributed? or are you just talking about the format?

Comment (KaelWD) [reply src/stories/Layout.vue]: The format, your indenting should be classed as a war crime

**Pull request merged**